src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS = 10;
+    private int index = 0;
+    private K[] keys = (K[]) new Object[MAX_ITEMS];
+    private V[] values = (V[]) new Object[MAX_ITEMS];
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < index; i++) {
+            if (checkIfEqual(key, keys[i])) {
+                values[i] = value;
+                return;
+            }
+        }
+        values[index] = value;
+        keys[index] = key;
+        index++;
     }
 
     @Override
     public V get(K key) {
+        int localIndex = 0;
+        for (K localKey : keys) {
+            if (checkIfEqual(key, localKey)) {
+                return values[localIndex];
+            }
+            localIndex++;
+        }
         return null;
     }
 
     @Override
     public int size() {
-        return -1;
+        return index;
+    }
+
+    private boolean checkIfEqual(K newKey, K localKey) {
+        return (localKey == newKey || localKey != null && localKey.equals(newKey));
```

Comment:
```suggestion
        return (localKey == newKey || (localKey != null && localKey.equals(newKey)));
```

It seems to me in this case, it is better to explicitly place the parentheses